chart/values.yaml

```
@@ -0,0 +1,78 @@
+# Default values for chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+app: liqo-dashboard
+replicaCount: 1
+namespace: liqo-dashboard
+host: liqo-dashboard.crownlabs.polito.it
+local: false
+
+image:
+  repository: lucarocco97
+  pullPolicy: Always
+  name: liqo-dashboard
+
+tls:
+- hosts:
+  - liqo-dashboard.crownlabs.polito.it
+  secretName: liqo-dashboard-app-certificate
+
+rbac:
+  create: true
+
+clusterRole:
+  roles:
+  - apiGroups: [""]
+    resources: ["nodes", "pods"]
+    verbs: ["get", "watch", "list"]
+  - apiGroups: ["discovery.liqo.io"]
+    resources: ["foreignclusters"]
+    verbs: ["get", "watch", "list"]
+  - apiGroups: ["sharing.liqo.io"]
+    resources: ["resourceoffers"]
+    verbs: ["get", "watch", "list"]
+  - apiGroups: ["metrics.k8s.io"]
+    resources: ["pods"]
+    verbs: ["get", "watch", "list"]
+
+deployments:
+  - name: fe
```

```suggestion
  - name: frontend
```

Nit. Easier to understand when seen for the first time. BTW, it is probably overkill to make these values configurable, while a real production ready chart should allow to tune much more aspects such as resources, security groups, ... (likely unnecessary in this case).